versions.props

```diff
@@ -30,10 +29,7 @@ org.projectnessie:* = 0.9.2
 org.junit.vintage:junit-vintage-engine = 5.7.2
 org.junit.jupiter:* = 5.7.2
 org.mockito:mockito-core = 3.7.7
-org.apache.hive:hive-exec = 2.3.8
-org.apache.hive:hive-service = 2.3.8
 org.apache.tez:tez-dag = 0.8.4
-org.apache.tez:tez-mapreduce = 0.8.4
```

Comment:
Why not do this with wildcards instead? Isn't this the same as setting `org.apache.hive:* = 2.3.8`?